### DISCUSSION TITLE
Get stuck with fownloading examples' data like brain.vtk

### DISCUSSION THREAD

**Discussion by zhengliuX** [pyvista/pyvista#1796] [Q&A] [answered]
Hi, I'm trying to run 
```
from pyvista import examples
examples.download_brain()
```
and I got stuck here for some network reasons. Briefly, I can't download the data, but I download vtk-data directly from github. Where should I put it? I'm using conda.

Thank you!

**Comment by tkoyama010** [comment, marked as answer]
We can simply read the 'brain.vtk'. This is same with using `download_brain` function.
```python
import pyvista as pv
dataset = pv.read('brain.vtk')
```

**Reply by zhengliuX** [reply]
Thanks a Lot!

**Comment by MatthewFlamm** [comment]
If you have the `vtk-data` repository downloaded, use this method:

https://docs.pyvista.org/extras/vtk_data.html

**Reply by zhengliuX** [reply]
Be of great help!!!
